packages/kbn-discover-contextual-components/kibana.jsonc

```
@@ -0,0 +1,5 @@
+{
+  "type": "shared-browser",
+  "id": "@kbn/discover-contextual-components",
+  "owner": "@elastic/obs-ux-logs-team"
```

Thanks for creating a separate package! Do you mind adding Discover team as a code owner?

```suggestion
  "owner": ["@elastic/obs-ux-logs-team", "@elastic/kibana-data-discovery"]
```